### PR TITLE
Enhancement: Allow installation of doctrine/dbal:^2.7.0 and doctrine/orm:^2.7.0

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] allows installation of `doctrine/dbal:^2.7.0` and `doctrine/orm:^2.7.0`

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -23,8 +23,8 @@
     },
     "require": {
         "php": ">=5.4.0",
-        "doctrine/orm": ">=2.2.1,<2.7-dev",
-        "doctrine/dbal": ">=2.2.1,<2.7-dev",
+        "doctrine/orm": ">=2.2.1,<3.0-dev",
+        "doctrine/dbal": ">=2.2.1,<2.8-dev",
```
Comment (localheinz) [inline composer.json]: For reference, see [`composer.json`](https://github.com/doctrine/dbal/blob/1f91c2d8bb18d6f439b2dea8cab5d58ca7468c77/composer.json#L39-L43) of `doctrine/dbal:dev-master`.

File: composer.json [hunk 2]
```diff
@@ -23,8 +23,8 @@
     },
     "require": {
         "php": ">=5.4.0",
-        "doctrine/orm": ">=2.2.1,<2.7-dev",
-        "doctrine/dbal": ">=2.2.1,<2.7-dev",
+        "doctrine/orm": ">=2.2.1,<3.0-dev",
```
Comment (localheinz) [inline composer.json]: For reference, see [`composer.json`](https://github.com/doctrine/doctrine2/blob/12b28571f55e4f04e3d480857fe79726d3e7dbb1/composer.json#L46-L50) of `doctrine/orm:dev-master`.

**Comment by glaubinix** [comment]
Thank you. Tagged as https://github.com/breerly/factory-girl-php/releases/tag/1.0.3

**Comment by localheinz** [comment]
Thank you, @glaubinix!

**Pull request merged**